### DISCUSSION TITLE
Alpha trend indicator suggestion...

### DISCUSSION THREAD

**Discussion by AykutCanturk** [DaveSkender/Stock.Indicators#779] [Ideas]
Can I suggest  adding Alpha Trned indicator from tradingview ? 

I say just take a look...

best....

**Comment by DaveSkender** [comment]
Is this just a traditional rolling Alpha?  Please send a link so I can see what you're recommending.
If you really want this in the library, help me out by providing more information about it and why it is exceptional.

[As a rule](https://github.com/DaveSkender/Stock.Indicators/discussions/1024), we also don't rely on publicly available code from other platforms.  They're not always a reliable source of truth.  We usually source recipes directly from original published works by reputable industry-recognized professionals.  Though, we've made exceptions to this rule when the works are open and the argument is compelling enough.

In any case, there are options to [create your own custom indicators](https://daveskender.github.io/Stock.Indicators/custom-indicators/#content) for things we decide to exclude from the main library.

**Reply by DaveSkender** [reply]
I have a simple workaround to get traditional Alpha in our Beta indicator documentation, see the [Pro Tips](https://daveskender.github.io/Stock.Indicators/indicators/Beta/#pro-tips) section; but admit this can be improved with a discrete Alpha indicator.

**Comment by AykutCanturk** [comment]
you check it from here. If you find it usefull then meybe you can consider it to add your library.

https://tr.tradingview.com/script/o50NYLAZ-AlphaTrend/

**Reply by DaveSkender** [reply]
I think this would be a good option for individuals to add to their own custom indicators.  I have to choose carefully what to add to this library because 1) I have a limited capacity to add all possible indicators, 2) I don't currently have a "more is better" strategy, and 3) I'm trying to stick to a core of time-tested reputable indicators.  If this one ends up being clearly more popular than all of the other 5,000+ indicators out there and is getting recognition by non-TradingView professionals in meaningful ways, I'll take another look.  It's too new and too random to add.

**Reply by AykutCanturk** [reply]
Okay, it was a suggestion. Thanks for this great library.
Best..

**Reply by DaveSkender** [reply]
Thanks for the suggestion though, I do appreciate it
